server/data/adjudicationsApiClient.ts

```diff
@@ -0,0 +1,23 @@
+import RestClient from './restClient'
+import { AdjudicationsApiClient } from './interfaces/adjudicationsApiClient'
+import { AdjudicationSummary } from '../interfaces/adjudicationSummary'
+import config from '../config'
+
+export default class AdjudicationsApiRestClient implements AdjudicationsApiClient {
+  constructor(private readonly restClient: RestClient) {}
+
+  private async get<T>(args: object, localMockData?: T): Promise<T> {
+    try {
+      return await this.restClient.get<T>(args)
+    } catch (error) {
+      if (config.localMockData === 'true' && localMockData) {
+        return localMockData
+      }
+      return error
+    }
+  }
+
+  async getAdjudications(bookingId: number): Promise<AdjudicationSummary> {
+    return this.get<AdjudicationSummary>({ path: `/adjudications/by-booking-id/${bookingId}` })
```

Comment:
[Here is this endpoint](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L102-L140) in the AdjudicationsAPI

I think we will need to add the `VIEW_ADJUDICATIONS` role to the prisoner profile client credentials as the [AdjudicationSummaryController has it configured](https://github.com/ministryofjustice/hmpps-manage-adjudications-api/blob/9e4fb75fe356845eda7c96c66aab998f792aecee/src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/controllers/AdjudicationSummaryController.kt#L29)